recipes/pytrends/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pytrends" %}
+{% set version = "4.7.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytrends-{{ version }}.tar.gz
+  sha256: 8ccb06c57c31fa157b978a0d810de7718ee46583d28cf818250d45f36abd2faa
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.3
+  run:
+    - lxml
+    - pandas >=0.25
+    - python >=3.3
+    - requests
+
+test:
+  imports:
+    - pytrends
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/dreyco676/pytrends
+  summary: Pseudo API for Google Trends
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
add comment:

```yaml
# license file manually packaged. See https://github.com/GeneralMills/pytrends/pull/443
license_file: LICENSE
```

I've opened a PR upstream to fix having to manually package the license file (https://github.com/GeneralMills/pytrends/pull/443). The comment in the recipe will be a visual reminder to check the status when you update the feedstock for future releases. Once the license file is included in the source distribution, you should delete the comment and remove the manually packaged copy.

Reply:
@synapticarbors Thanks for the review - added the comment! Could you take another look please?